### PR TITLE
#6 - Create Mapper (Backend)

### LINKED ISSUE #6: Create Mapper (Backend)

### PR DESCRIPTION
**Pull request by NN171**
Created mappers for every entity class. Added properties to application.properties

### CODE REVIEW

File: backend/BackendMIIT/src/main/resources/application.properties [hunk 1]
```diff
@@ -1 +1,6 @@
 spring.application.name=BackendMIIT
+spring.jpa.hibernate.ddl-auto=update
+spring.datasource.url=jdbc:postgresql://localhost:5432/MiitInside
+spring.datasource.username=postgres
+spring.datasource.password=postgres
+spring.datasource.driver-class-name=org.postgresql.Driver
```
Comment (ImmortalIdiot) [inline backend/BackendMIIT/src/main/resources/application.properties]: Add the empty line in the end of the file :D

File: backend/BackendMIIT/pom.xml [hunk 2]
```diff
@@ -47,13 +49,20 @@
...
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.mapstruct</groupId>
+			<artifactId>mapstruct</artifactId>
+			<version>${mapstruct.version}</version>
+		</dependency>
+
```
Comment (ImmortalIdiot) [inline backend/BackendMIIT/pom.xml]: Extra empty line

**Comment by ImmortalIdiot** [comment]
After resolving my comments, write feedbacks. And if the solution is ok - i will mark that the comment resolved. If all comments had been resolved - I will aprove the pr

File: backend/BackendMIIT/src/main/resources/application.properties (hunk 1 above)
Comment (NN171) [reply backend/BackendMIIT/src/main/resources/application.properties]: Added

File: backend/BackendMIIT/pom.xml (hunk 2 above)
Comment (NN171) [reply backend/BackendMIIT/pom.xml]: Added new empty lines

**Comment by ImmortalIdiot** [comment]
Nice job

**Pull request merged**